core/templates/components/summary-tile/subtopic-summary-tile.directive.ts

```diff
@@ -65,3 +64,18 @@ angular.module('oppia').directive('subtopicSummaryTile', [
       ]
     };
   }]);
+
+import { Directive, ElementRef, Injector, Input } from '@angular/core';
+import { UpgradeComponent } from '@angular/upgrade/static';
+
+@Directive({
+  selector: 'subtopic-summary-tile'
+})
+export class SubtopicSummaryTileDirective extends UpgradeComponent {
+  @Input() subtopic;
```

Comment:
I see the types are defined elsewhere, why are they not added here?

Reply:
This is still angular js. When this directive is migrated to angular completely, that's when the types will be added.

Reply:
Makes sense. Thanks